user-org-service/app/utils/validator/schema/SchemaManager.java

```diff
@@ -0,0 +1,92 @@
+package utils.validator.schema;
+
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.loader.SchemaClient;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.sunbird.util.LoggerEnum;
+import org.sunbird.util.ProjectLogger;
+
+public class SchemaManager {
+
+  private static Map<String, SchemaMapper> schemaInfoMap = new HashMap<>();
+  private static final String schemaMapperFile = "schemaMapper.properties";
+
+  public SchemaManager() {}
+
+  private Properties getSchemaRoutesFileMap() {
+    Properties prop = new Properties();
+    try (InputStream inputStream =
+        this.getClass().getClassLoader().getResourceAsStream(schemaMapperFile)) {
+      prop.load(inputStream);
+      ProjectLogger.log(
+          String.format(
+              "%s:%s: Properties Populated",
+              this.getClass().getSimpleName(), "getSchemaRoutesFileMap"),
+          LoggerEnum.DEBUG.name());
+      return prop;
+    } catch (Exception e) {
+      ProjectLogger.log(
+          String.format(
+              "%s:%s: error occurred while loading schemaMapperFile",
+              this.getClass().getSimpleName(), "getSchemaRoutesFileMap"),
+          LoggerEnum.DEBUG.name());
+      System.exit(-1);
+    }
+    return prop;
+  }
+
+  /** this method is used to init the schemas and save them in cache map name schemaInfoMap */
+  public void initSchemas() {
```

Comment:
Are we caching all available schema object at server start up or doing when needed?

Reply:
on server startup sir

Reply:
ok